pkg/nsx/services/vpc/store_test.go

```diff
@@ -162,7 +162,7 @@ func TestVPCStore_CRUDResource(t *testing.T) {
 }
 
 func TestVPCStore_CRUDResource_List(t *testing.T) {
-	vpcCacheIndexer := cache.NewIndexer(keyFunc, cache.Indexers{common.TagScopeVPCCRUID: indexFunc})
+	vpcCacheIndexer := cache.NewIndexer(keyFunc, cache.Indexers{common.TagScopeNetworkInfoCRUID: indexFunc})
```

Comment:
remove this indexer too, we may need to carefully examine the code which used this indexer.